apps/src/templates/DataWorkspace.jsx

```diff
@@ -0,0 +1,10 @@
+var React = require('react');
+
+var DataWorkspace = React.createClass({
+  render: function () {
+    return <div id="dataWorkspaceWrapper">
+      Data workspace
+    </div>;
```

Comment:
nit: Prefer having open/close tags align with each other in terms of indentation, so I'd do something like:
```jsx
return (
  <div id="dataWorkspaceWrapper">
    Data workspace
  </div>
);